ddtrace/tracer/writer_test.go

```diff
@@ -275,6 +275,21 @@ func TestLogWriterOverflow(t *testing.T) {
 		assert.Equal(io.EOF, err)
 	})
 }
+func TestJSONEncodeSpanNewLines(t *testing.T) {
+	assert := assert.New(t)
+	s := newSpan("name", "srv", "res", 2, 1, 3)
+	s.Start = 12
+	s.Meta["query\n"] = "Select * from \n Where\nvalue"
+	s.Metrics["version\n"] = 3
+
+	h := &logTraceWriter{}
```

Comment:
```suggestion
	var w logTraceWriter
```
Not sure what `h` was. `w` for writer maybe is better?